_posts/2016-06-16-cold-vs-hot-observables.md

```diff
@@ -383,5 +383,11 @@ this.contacts = http.get('contacts.json')
 {% endraw %}
 {% endhighlight %}
 
+**Note: When using multiple async pipes on streams with default values, the `.share()` operator might cause problems:**
+
+The `share()`will publish the first value of the stream on the first subscription. The first async pipe will trigger that subscription and get that initial value. The second async pipe however will subscribe after that value has already been emitted and therefore miss that value.
```

Comment:
I'd argue that this would not be a problem in practice most of the time because both subscriptions will be in the same tick (let's say tick zero) whereas the first emission will most likely arrive asynchronous. That means, as long as the first emission isn't synchronous there would not be a real world problem. 

Your example uses an `BehaviourSubject` with an initial *synchronous* value so it can demonstrate the problem. 

What is the default `BufferSize` for replay? You most likely don't want to keep every message of the stream in the buffer for future subscribers to replay.